include/vcpkg/base/messages.h

```diff
@@ -2091,5 +2091,168 @@ namespace vcpkg
+                    "Restricted headers paths are present. These files can prevent the core C++ runtime and "
+                    "other packages from compiling correctly:");
+    DECLARE_MESSAGE(PortBugAllowRestrictedHeaders,
+                    (msg::env_var),
+                    "",
+                    "In exceptional circumstances, this policy can be disabled via {env_var}");
+    DECLARE_MESSAGE(PortBugDuplicateIncludeFiles,
+                    (),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "Include files should not be duplicated into the /debug/include directory. If this cannot "
+                    "be disabled in the project cmake, use\n"
+                    "file(REMOVE_RECURSE \"${CURRENT_PACKAGES_DIR}/debug/include\")");
+    DECLARE_MESSAGE(PortBugDebugShareDir,
+                    (),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "/debug/share should not exist. Please reorganize any important files, then use\n"
+                    "file(REMOVE_RECURSE \"${CURRENT_PACKAGES_DIR}/debug/share\")");
+    DECLARE_MESSAGE(PortBugMissingFile,
+                    (msg::path),
+                    "",
+                    "The /{path} file does not exist. This file must exist for CMake helper ports.");
+    DECLARE_MESSAGE(PortBugMergeLibCMakeDir,
+                    (msg::spec),
+                    "",
+                    "The /lib/cmake folder should be merged with /debug/lib/cmake and moved to "
+                    "/share/{spec}/cmake.");
+    DECLARE_MESSAGE(PortBugUseVcpkgCMakeConfigFixup,
+                    (),
+                    "",
+                    "Please use the helper function `vcpkg_cmake_config_fixup()` "
+                    "from the port vcpkg-cmake-config.`");
+    DECLARE_MESSAGE(
+        PortBugMisplacedCMakeFiles,
+        (msg::spec),
+        "",
+        "The following cmake files were found outside /share/{spec}. Please place cmake files in /share/{spec}.");
+    DECLARE_MESSAGE(PortBugDllInLibDir,
+                    (),
+                    "",
+                    "The following dlls were found in /lib or /debug/lib. Please move them to /bin or "
+                    "/debug/bin, respectively.");
+    DECLARE_MESSAGE(PortBugMissingLicense,
+                    (msg::spec),
+                    "'{CURRENT_PACKAGES_DIR}' should not be translated.",
+                    "The software license must be available at ${CURRENT_PACKAGES_DIR}/share/{spec}/copyright");
+    DECLARE_MESSAGE(PortBugFoundCopyrightFiles, (), "", "The following files are potential copyright files:");
+    DECLARE_MESSAGE(PortBugFoundExeInBinDir,
+                    (),
+                    "",
+                    "The following EXEs were found in /bin or /debug/bin. EXEs are not valid distribution targets.");
+    DECLARE_MESSAGE(PortBugDllsWithNoExports, (), "", "The following DLLs have no exports:");
```

Comment:
```suggestion
    DECLARE_MESSAGE(PortBugDllsWithNoExports, (), "'exports' means an entry in a DLL's export table", "The following DLLs have no exports:");
```

I think we should clarify that this doesn't mean export as in the trade sense for translators. Ditto elsewhere.